models/core-java/src/main/java/org/commonjava/indy/stats/IndyVersioning.java

```diff
@@ -28,6 +28,12 @@
 @Named
 public class IndyVersioning
 {
+    public final static String HEADER_INDY_CLIENT_API = "Indy-Client-API";
```

Comment:
Not sure this is the proper class for those. This class is just for controlling indy's api version. Maybe use somewhere else for trace related constants?

Reply:
@ruhan1 Agree, move them to ClientMetricManager in commit amending.